app/utils/computed-helpers.js

```diff
@@ -16,17 +16,19 @@ export const computedSegmentedLink = function(property) {
       const splitted = this.get(property) ? this.get(property).split('://') : [];
       if (!splitted || splitted.length === 0 || (splitted.length === 1 && splitted[0].includes('http'))) {
         return {
-          protocol : 'https',
+          protocol : 'https://',
           address  : ''
         };
       }
+      const socialUrl = splitted[1].split('/');
       return {
-        protocol : splitted[0],
-        address  : splitted[1]
+        protocol : `${splitted[0]}://${socialUrl[0]}/`,
+        address  : socialUrl[1]
       };
     },
     set(key, value) {
-      const finalLink = values(value).join('://');
+      const finalLink = values(value).join('');
+      
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6103967)